doc/source/cluster/kubernetes/user-guides/kubectl-dashboard.md

```diff
@@ -0,0 +1,33 @@
+(kubectl-dashboard)=
+
+# Use kubectl dashboard (alpha)
+
+Starting from KubeRay v1.4.0, you can use the open source dashboard UI for KubeRay.
+
```

Comment:
Add a sentence to explain the current status (still experimental not ready for production, but feedbacks are welcome).